pancancer_classifier.py

```diff
@@ -500,12 +453,13 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
                                       genes=alt_genes, loss_df=copy_loss_df,
                                       gain_df=copy_gain_df)
     # Append tissue id
-    y_alt = y_alt.assign(disease=clinical_df['acronym'])
```

Comment:
> it's a slice of mutation_df that has the alternative genes classifier applied?

exactly. `mutation_df` has info for all ~20,000 genes. the "original" `y` splits off this matrix to only the input genes of interest. `y_alt` will generate a new Y matrix (used for machine learning classification) but the same classifier will be applied. In my examples, I use a completely different X matrix for both examples, but theoretically, the same exact X can be used (which is a pretty strange concept)